CopyPaste.cs

```diff
@@ -1658,7 +1680,7 @@ private void PasteEntity(Dictionary<string, object> data, PasteData pasteData, B
 
                 if (autoTurret != null)
                 {
-                    autoTurret.Invoke(autoTurret.UpdateAttachedWeapon, 0.5f);
+                    autoTurret.UpdateAttachedWeapon();
```

Comment:
Updating the attached weapon immediately is now required so the child weapon prefabs are spawned before processing the child entities in `data["children"]`.